src/traces/bar/attributes.js

```diff
@@ -35,8 +42,32 @@ module.exports = {
     y: scatterAttrs.y,
     y0: scatterAttrs.y0,
     dy: scatterAttrs.dy,
+
     text: scatterAttrs.text,
 
+    textposition: {
```

Comment:
>  I think this is doable

Great. That's all I wanted to here for now. No action required in this iteration.